app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
Well true, that's for Angular2. I don't think the first point is invalid for Angular 1 either?

The naming convention doesn't work exactly like that, I thought the suffix was meant to be the type for the class, not for replacing camel case?